README

```diff
@@ -1,10 +1,52 @@
+# Dylan - a language grammar for highlight.js 
+
 This repository contains highlight.js support for the Dylan language.
 
 Dylan is a multi-paradigm functional and object-oriented programming
 language. It is dynamic while providing a programming model designed
 to support efficient machine code generation, including fine-grained
 control over dynamic and static behaviors.
 
-See https://github.com/highlightjs/highlight.js
-and https://opendylan.org/
+## Usage
+
+Include the Highlight.js library in your webpage or Node app, then load this module.
+
+### Static website or simple usage
+
+Load the module after loading Highlight.js.  You'll use the minified version found in the `dist` directory.  This module is just a CDN build of the language, so it will register itself as the Javascript is loaded.
+
+```html
+<script type="text/javascript" src="/path/to/highlight.min.js"></script>
+<script type="text/javascript" charset="UTF-8"
+  src="/path/to/highlightjs-dylan/dist/dylan.min.js"></script>
+<script type="text/javascript">
+  hljs.initHighlightingOnLoad();
+</script>
+```
+
+### With Node or another build system
+
+If you're using Node / Webpack / Rollup / Browserify, etc, require the language module, then register it with Highlight.js.
+
+```javascript
+var hljs = require('highlightjs');
+var hljsRobotsTxt = require('highlightjs-dylan');
```

Comment:
Might want to update this variable name.

Reply:
Sorry more questions - as I understand it, 'highlightjs' is deprecated and it should be 
```
var hljs = require('highlight.js');
```
instead?
(As you noticed, I just copied this from the robots.txt repo!)

Reply:
Correct, I'll go update that!

Reply:
OK, changed it here. I am aware this refers to "highlightjs-dylan" and not "highlight-dylan" but this will become correct when #4 is fixed.